README.md

```diff
@@ -101,6 +99,12 @@ postgresql_server_install 'Setup my postgresql 9.5 server' do
 end
 ```
 
+#### Known issues
+
+On Debian and Ubuntu platforms, `initdb_locale` should be set to `en_US.utf8`
```

Comment:
I'm not sure this is debian/ubuntu specific. As in Docker/Dokken UTF-8 is the "standard" 

something with more generic might be better here :)